code/game/machinery/vending.dm

```diff
@@ -218,6 +220,11 @@
 	if(!user.unEquip(W))
 		return
 
+	for(var/a in blacklisted)
+		if(istype(W, a))
+			to_chat(user, SPAN_NOTICE("You are unable to add this product to the receptor."))
```

Comment:
A receptor/acceptor in vending machines is for the bills. This phrase would imply you add an item through the credit bill slot. The part where you put the item you want to sell/buy is called the product shelf.

The phrase should be "You are unable to add this to the product shelves."

Reply:
This is accually same message as the one that rejects items?

Reply:
`to_chat(user, SPAN_NOTICE("You insert \the [W] in the product receptor."))`

Reply:
Well then its wrong as well.